akka-docs/rst/java/stream/stages-overview.rst

```diff
@@ -894,7 +894,7 @@ Similar to ``Flow.fromSinkAndSource`` however couples the termination of these t
 E.g. if the emitted ``Flow`` gets a cancellation, the ``Source`` of course is cancelled,
 however the Sink will also be completed. The table below illustrates the effects in detail:
 
-+-------------------------------------------------+-----------------------------+---------------------------------+
++=================================================+=============================+=================================+
```

Comment:
Ah, I was wondering why it was complaining about that earlier but couldn't figure out what was wrong with the table. 👍